src/contact/style.css

```diff
@@ -0,0 +1,145 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+  font-family: 'Mansalva', cursive;
+}
+
+body {
+  width: 100%;
+  height: 100%;
+  background: black;
+}
+
+.main-container {
+  width: 100%;
+  display: flex;
+  flex-flow: column;
+  color: white;
+  background: rgb(0, 0, 0);
+}
+
+.navbar {
+  display: flex;
+  flex-flow: row;
+  justify-content: flex-end;
+  width: 100%;
+  background-color: black;
+  font-size: .75rem;
+}
+
+.navlinks {
+  width: 100%;
+  display: flex;
+  flex-flow: row wrap;
+  text-align: center;
+  align-items: center;
+  color: white;
+}
+
+.navlinks li {
+  width: 18%;
+  list-style-type: none;
+}
+
+.main-container {
+  width: 100%;
+}
+
+.main {
+  width: 100%;
+  display: flex;
+  flex-flow: column;
+  justify-content: center;
+  margin-top: 20px;
```

Comment:
since this element's parent is using flex, you can add `flex: 1;` to the main page so it takes up the rest of the space after the nav height's.

Reply:
and you can add `align-items: center;` to center the child form element.